.github/workflows/main.yml

```diff
@@ -0,0 +1,42 @@
+name: testcontainers-python
+on:
+  - push
```

Comment:
Do we need something for pull requests here, e.g. something like https://github.com/tillahoffmann/localscope/blob/1584b66ce2e1cd02aa10b82332b5775f6ebcbdb7/.github/workflows/build.yml#L6-L12?

Or is that already covered?

Reply:
no, it wasn't covered. I did a mirror translation from TravisCI file to Github action file.

Added `pull_request` trigger action.